src/devices/GrovePi/Models/PinMode.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace Iot.Device.GrovePiDevice.Models
+{
+    /// <summary>
+    /// Defines the pin mode, either input, either output
+    /// </summary>
+    public enum PinMode
```

Comment:
why not use `PinMode` from Gpio?

Reply:
Yes, same for PinLevel (PinValue). They are actually the same values in the enum. So all will work the same way